### ISSUE TITLE
Hi Danee. Great work supporting this app for Homey and Ring.

As part of my ecosystem I have 2 stickup cams outside, allowing me to extend the siren from the inside to the outside as well when nobody is home. 

As part of the solution I also implemented a flow that should stop the sirens upon disarmed mode, so the sirens stop as soon as possible in case of a self-inflicted alarm for instance.

But when I apply the Siren Off step, the sirens will in effect start the siren if not activated, or extend the activated siren without directly stopping. It will run for about 30 seconds? 

Is this by design or maybe a bug?

Thank you for looking into it.

Keep up the much appreciated work. 

### ISSUE THREAD

**Issue by Edwin-VC** [daneedk/com.amazon.ring#8]

**Comment by daneedk** [comment]
Hi Edwin,

Seems I've missed this report, even though it's hard to overlook ;)

Best place to get support is here: https://community.homey.app/t/ring-doorbell-and-camera-support-for-homey/76364

I'm not aware of a bug regarding switching of the sirene, I will look into it.

**Comment by daneedk** [comment]
Hi Edwin,

I do think I found something, when first implementing I had to rely on others to test this as I had no Ring Camera device at that time and I never got a mention the Siren of didn't work. I now do have a camera device and was able to test it myself and found out it indeed didn't work as expected. I was able to easily fix it however and have already published a new version to the Homey App Store. It should automatically be installed on your Homey when you have auto updates for the app enabled.
